docs/source/theming/using-third-party-themes.md

```diff
@@ -35,7 +35,7 @@ An example of this behavior is the blocks view: you are in a public view because
 
 ## Setting up the theme
 
-In your Volto project, customize the file `src/theme.js`:
+In your Volto project, customize the file `src/theme.js`, creating a `customization/@root/theme.js` file:
```

Comment:
This sentence doesn't make sense to me. How do I customize one file by creating another file?

Reply:
This is closer, however the subsequent diff code-block doesn't make sense for what should go into `customizations/@root/theme.js`.

```suggestion
Customize Volto's {file}`src/theme.js` by creating a file in the {file}`customizations` folder to override the original.
Create the file {file}`customizations/@root/theme.js`.
```

Reply:
The contents here and in the `add-ons/theme.md` file should be the same, as the procedure is the same. So if you think that the first change is clear enough, we can copy it here.

Reply:
PR #6784 merged!